core/src/main/kotlin/org/evomaster/core/search/gene/ArrayGene.kt

```diff
@@ -50,13 +50,17 @@ class ArrayGene<T>(
         }
 
         if (minSize != null && maxSize != null && minSize!! > maxSize!!){
-            throw IllegalArgumentException(
-                "ArrayGene "+name+": minSize (${minSize}) is greater than maxSize ($maxSize)")
+//            throw IllegalArgumentException(
+//                "ArrayGene "+name+": minSize (${minSize}) is greater than maxSize ($maxSize)")
+            log.warn("ArrayGene "+name+": minSize (${minSize}) is greater than maxSize ($maxSize)")
         }
 
         if (maxSize != null && elements.size > maxSize!!) {
-            throw IllegalArgumentException(
-                "ArrayGene "+name+": More elements (${elements.size}) than allowed ($maxSize)")
+//            throw IllegalArgumentException(
+//                "ArrayGene "+name+": More elements (${elements.size}) than allowed ($maxSize)")
+
+            log.warn("ArrayGene "+name+": More elements (${elements.size}) than allowed ($maxSize)")
+
         }
```

Comment:
not sure about this...
if it happens it means it is a bug in EM, which we must fix.
if we rather log it instead of throwing an exception, we might just miss this issue.